specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json

```diff
@@ -15,6 +15,527 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PackageTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/ArchiveNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource does not exist, 404 (Not found) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveGet": {
+            "$ref": "./examples/ArchiveGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Archives"
+        ],
+        "description": "Creates a archive for a container registry with the specified parameters.",
+        "operationId": "Archives_Create",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PackageTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/ArchiveNameParameter"
+          },
+          {
+            "name": "archiveCreateParameters",
+            "in": "body",
+            "description": "The parameters for creating a archive.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the operation has completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            }
+          },
+          "201": {
+            "description": "The request was successful; the operation will complete asynchronously.",
```

Comment:
>"The request was successful; the operation will complete asynchronously." [](http://example.com/codeflow?start=27&length=73)

"ARM create operation completed successfully."

Reply:
The description of the response code 200 is same for all the resources.